deployment/conf/.templates/deployment.cfg.templ

```diff
@@ -1,9 +1,11 @@
 [Workspace]
 mongodb-database={{ default .Env.mongodb_database "workspace" }}
+mongodb-host={{ default .Env.mongodb_host "ci-mongo" }}
+mongodb-user={{ default .Env.mongodb_user "wsserv" }}
+mongodb-pwd={{ default .Env.mongodb_pwd "" }}
```

Comment:
these haven't changed, just moved all the mongo config to be in the same place

Reply:
:+1: